### ISSUE TITLE
[FR] Show only remaining time on Ender 3 V2 DWIN_MARLINUI

### ISSUE THREAD

**Issue by RustedAperture** [MarlinFirmware/Marlin#25135]
### Did you test the latest `bugfix-2.1.x` code?

Yes, and the problem still exists.

### Bug Description

When disabling elapsed time by commenting it out, it still shows on the ender 3 v2 display. Have not tested with other displays.

I have enabled only remaining time but elapsed still shows.

### Bug Timeline

not sure

### Expected behavior

only remaining time to show

### Actual behavior

cycles between remaining and elapsed

### Steps to Reproduce

1. comment out `#define SHOW_ELAPSED_TIME`
2. remove comment from `#define SHOW_REMAINING_TIME`

### Version of Marlin Firmware

2022-12-21

### Printer model

Ender 3 V2

### Electronics

SKR Mini E3 V2, Stock Ender 3 screen

### Add-ons

BLTouch

### Bed Leveling

UBL Bilinear mesh

### Your Slicer

Prusa Slicer

### Host Software

OctoPrint

### Don't forget to include

- [X] A ZIP file containing your `Configuration.h` and `Configuration_adv.h`.

### Additional information & file uploads

[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/10283896/Marlin.zip)

**Comment by RustedAperture** [comment]
Initialized the EEPROM and the problem was solved, however, now the LCD is not usable during the print.

**Comment by RustedAperture** [comment]
Bump

**Comment by ellensp** [comment]
running DWIN_MARLINUI_PORTRAIT on Stock Ender 3 **V2** screen

**Comment by ellensp** [comment]
looking at https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.1.x/Marlin/src/lcd/e3v2/marlinui/ui_status_480x272.cpp#L369-L422

From looking at the code this is simply not been implemented yet

**Comment by RustedAperture** [comment]
> running DWIN_MARLINUI_PORTRAIT on Stock Ender 3 **V2** screen

Yes, sorry should have specified since there are multiple available for the DWIN E3V2 screen.

**Comment by RustedAperture** [comment]
> looking at https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.1.x/Marlin/src/lcd/e3v2/marlinui/ui_status_480x272.cpp#L369-L422
> 
> From looking at the code this is simply not been implemented yet

Is it ok to change the title to [FR] then?

**Comment by ellensp** [comment]
Yes I think that would be appropriate.

**Comment by thinkyhead** [comment]
Give PR 28449 a test to see if it works. If there are issues then report them on the PR itself.
